tests/test_utils.py

```
@@ -1,44 +1,28 @@
+
+def test_extract_price_comma_thousands() -> None:
+    # US/UK format with comma thousands separator must be preserved unchanged.
+    assert extract_price("$1,875.00") == "$1,875.00"
+    assert extract_price("$999") == "$999"
+
+
+def test_extract_price_space_thousands() -> None:
+    # French/European format uses a (non-breaking) space as thousands separator.
+    # Regression test: previously "1 875" was split into "1 | 875".
+    assert extract_price("1 875 C$") == "1875"
+    assert extract_price("1 875 C$") == "1875"  # non-breaking space
```

Ruff (0.16.3)

[warning] String contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF001)